mmdet/models/utils/activation.py

```diff
@@ -0,0 +1,33 @@
+import torch.nn as nn
+
+activation_cfg = {
+    # format: layer_type: (abbreviation, module)
+    'relu': nn.ReLU,
+    'leaky_relu': nn.LeakyReLU,
+}
+
+
+def build_activation_layer(cfg):
+    """ Build activation layer
+
+    Args:
+        cfg (dict): cfg should contain:
+            type (str): Identify activation layer type.
```

Comment:
Additional indents.

Reply:
The indents are for the key-value in `cfg`.